n26/library/templates/authoring/ingest_preview.html

```
@@ -0,0 +1,202 @@
+                    {% for diff in preview.diffs %}
+                        <tr>
+                            <td>
+                                <div class="flex items-start gap-2">
+                                    <span class="text-muted tabular-nums">×{{ diff.count }}</span>
+                                    <span class="text-ink-900 dark:text-ink-100">{{ diff.shape }}</span>
+                                </div>
+                                <ul class="mt-1 space-y-0.5">
+                                    {% for example in diff.examples %}
+                                        <li class="text-sm text-muted">{{ example.name }} — {{ example.said }}</li>
+                                    {% endfor %}
+                                    {% if diff.count > diff.examples|length %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)